docs/examples/vault-default-user/rabbitmq.yaml

```diff
@@ -11,8 +11,9 @@ spec:
       # https://www.vaultproject.io/docs/platform/k8s/injector/annotations
       annotations:
         vault.hashicorp.com/template-static-secret-render-interval: "15s"
-      pathDefaultUser: secret/data/rabbitmq/config
-    # Optionally, deploy a sidecar container that rotates admin password in RabbitMQ server
-    # whenever Vault sidecar container puts a new password into
-    # /etc/rabbitmq/conf.d/11-default_user.conf
-    credentialUpdaterImage: "rabbitmqoperator/admin-password-updater:0.1.1"
+      defaultUserPath: secret/data/rabbitmq/config
+      # Optionally, override the sidecar container image that rotates admin password in RabbitMQ server
+      # whenever Vault sidecar container puts a new password into
+      # If not specified, it defaults to "rabbitmqoperator/admin-password-updater:0.1.1"
+      # /etc/rabbitmq/conf.d/11-default_user.conf
```

Comment:
The ordering of the lines is not correct.

Also, let's not define the default version here because we don't want to update the default image version in the examples whenever the image version changes.

Reply:
Fixed in https://github.com/rabbitmq/cluster-operator/pull/846/commits/9b9a3d0d61f0ed838b279ac767dcdbbdc236a0b2